src/apps/appstore/views/marketConnectorSettings.html

```diff
@@ -37,6 +47,26 @@
 									<span class="help-block">{{ i18n.marketplace.edition.helpApiUrl }}</span>
 								</div>
 							</div>
+							<div class="control-group">
+								<label class="control-label">{{ i18n.marketplace.edition.kazooAllInOneLabel }}</label>
+								<div class="controls">
+								{{#monsterSwitch}}
+								<input type="checkbox" name="is_aio_cluster" id="all_in_one" checked="{{ config.is_aio_cluster}}"/>
```

Comment:
This doesn't seems to work, I set it is_aio_cluster to false and it still shows toggle as true